Source/Csla/Core/IContextManager.cs

```diff
@@ -41,7 +42,7 @@ public interface IContextManager
     /// <summary>
     /// Gets the local context.
     /// </summary>
-    IContextDictionary GetLocalContext();
+    IContextDictionary? GetLocalContext();
```

Comment:
add `#nullable enable` to avoid build warnings for everyone else while this is all being worked on. We can remove that later when the conversion is done.

Reply:
Yep good idea.